packages/styles/globals.import.styl

```diff
@@ -4,7 +4,7 @@ body
 
 body
   font-family inherit
-  > div
+  id['__flow-root']
```

Comment:
Ah I see. It would be better not to depend on flow layout in here, so I guess I'd prefer the first child. But a different layout package might structure things differently and require changes anyway.